README.md

```diff
@@ -1402,6 +1402,36 @@ Magic numbers should be avoided as they often lack documentation.
 Forcing them to be stored in variables gives them implicit documentation.
 With [no-magic-numbers](https://palantir.github.io/tslint/rules/no-magic-numbers/) lint rule, we make code more readable and refactoring easier by ensuring that special numbers are declared as constants to make their meaning explicit.
 
+<details><summary>🖊 <b>Code Examples</b></summary>
+
+### 👍 Doing It Right Example: Do not hard code the value of PI (3.14159) inside a function
+
+```typescript
+function calculateCircleArea(radius: number) {
+  return 3.14159 * radius * radius;
```

Comment:
Looks like we swapped the examples :smile:

Reply:
Oops. Fixed